ChineseFilter/sections/specific.txt

```diff
@@ -74,4 +74,8 @@ nfmovies.com##a[href][onclick^="openurl()"] img[src^="/static/"]
 !+ PLATFORM(ios, ext_safari, ext_android_cb)
 nfmovies.com##body div[id^="aaaDiv"]
 !+ PLATFORM(windows, mac, android)
-||nfmovies.com/static/side.jpg$image,redirect=nooptext,domain=nfmovies.com
\ No newline at end of file
+||nfmovies.com/static/side.jpg$image,redirect=nooptext,domain=nfmovies.com
+||cocomanhua.com/sw.js$script
```

Comment:
The request has been blocking already.

Reply:
ok, marked the change

Reply:
wait, this rule (||cocomanhua.com/sw.js$script) is already block? cause when i disable this rule from user rule, one of the ads came back

Reply:
If you are going to say easylist block it alr using the rule below, it is also not working

/^https?:\/\/.*\/.*(sw[0-9a-z._-]|\.notify\.).*/$script,domain=1337x.to|1newmoviesonline.me|9xflix.asia|anonfiles.com|aparat.cam|bdupload.asia|bitfly.io|cashurl.in|clipwatching.com|clk.ink|cloudvideo.tv|cocomanhua.com|cutewallpaper.org|downloadpirate.com|eztv.io|eztv.tf|eztv.yt|filmix.co|filmlinks4u.is|flashx.pw|fmovies.taxi|fmovies.world|freeroms.com|fullmatchesandshows.com|getlink.pro|gomoviesfree.sc|heapfiles.com|indishare.org|intoupload.net|limetorrents.info|ling.online|linksly.co|macupload.net|mangareader.net|megaup.net|mhktricks.org|mixdrop.co|monova.org|monova.to|newser.com|nutritioninsight.com|ouo.press|piknu.com|powv1deo.cc|powvideo.net|powvldeo.cc|powvldeo.co|primewire.sc|putlocker.fyi|putlocker.style|putlockers.movie|readcomiconline.to|realgfporn.com|sendit.cloud|sendspace.com|serials.ws|shrinke.me|shrinkhere.xyz|solarmovie.to|techrfour.com|thatav.net|thewpsite.com|torlock.com|torrentdownload.info|uiz.io|up-load.io|upload.ac|uploadever.com|uploadproper.net|uploadrive.com|uplovd.com|upstream.to|uptobox.com|userscloud.com|v96-surf.com|vev.io|vev.red|vidbull.tv|videobin.co|vidlox.me|vidop.icu|vidoza.co|vidoza.net|vidtomp3.com|vidtoply.com|vidup.io|vumoo.life|watchtvseries.video|xnxxxvideos.co|xtgem.com|xtits.com|y2mate.com|yify-movies.net|yourbittorrent2.com|youvideoporno.mobi|youzik.com|zippyshare.com

Reply:
Yes, it's blocked by regex rule. Unfortunately, some of them may not work on iOS and I can't test it because I don't have a device.
So,`||cocomanhua.com/sw.js` does it work well on iOS?

Reply:
||cocomanhua.com/sw.js work for IOS

Reply:
Ok and what about exception? Does it need for iOS?

Reply:
exception? Do you mean the other domain?

Reply:
This one `@@||cocomanhua.com/js/ad_/*.js`

Reply:
Yup, this also work for IOS

Reply:
```
@@||cocomanhua.com/js/ad_/*.js
||cocomanhua.com/sw.js
```
Are they enough for iOS?

Reply:
Yup, with this 2 rule, it should fix cocomanhua anti adblock problem for ios user

Reply:
Ok, I appreciate your help. Added them to the AdGuard Chinese directly:)